### PR TITLE
Make the OSSL_CMP manual conform with man-pages(7)

### PR DESCRIPTION
**Pull request by levitte**
Details from man-pages(7) that are used:

    Formatting conventions for manual pages describing functions

        ...
        Variable names should, like argument names, be specified in italics.
        ...

    Formatting conventions (general)

        ...
        Special macros, which are usually in uppercase, are in bold.
        Exception: don't boldface NULL.
        ...

### CODE REVIEW

File: doc/man3/OSSL_CMP_ITAV_set0.pod [hunk 1]
```diff
@@ -29,21 +29,21 @@ ITAV is short for InfoTypeAndValue. This type is defined in RFC 4210
 section 5.3.19 and Appendix F. It is used at various places in CMP messages,
 e.g., in the generalInfo PKIHeader field, to hold a key-value pair.
 
-OSSL_CMP_ITAV_create() creates a new OSSL_CMP_ITAV structure and fills it in.
-It combines B<OSSL_CMP_ITAV_new()> and B<OSSL_CMP_ITAV_set0>.
+OSSL_CMP_ITAV_create() creates a new B<OSSL_CMP_ITAV> structure and fills it in.
```
Comment (t8m) [inline doc/man3/OSSL_CMP_ITAV_set0.pod:32]: Here the OSSL_CMP_ITAV is in bold but elsewhere in the document there are other typedefs like OSSL_CMP_CTX and ASN1_TYPE kept in regular font.

File: doc/man3/OSSL_CMP_log_open.pod [hunk 2]
```diff
@@ -92,14 +92,15 @@ any pending CMP-specific log output and deallocate related resources.
...
+I<component> usually is a function or module name.
 If it is NULL, empty, or "(unknown function)" then "CMP" is used as fallback.
 
 OSSL_CMP_print_errors_cb() outputs any entries in the OpenSSL error queue.
-It is similar to B<ERR_print_errors_cb()> but uses the CMP log callback function
-C<log_fn> for uniformity with CMP logging if not B<NULL>. Otherwise it prints to
-STDERR using B<OSSL_CMP_print_to_bio(3)> (unless OPENSSL_NO_STDIO is defined).
+It is similar to L<ERR_print_errors_cb(3)> but uses the CMP log callback
+function I<log_fn> for uniformity with CMP logging if not NULL. Otherwise it
+prints to STDERR using L<OSSL_CMP_print_to_bio(3)> (unless OPENSSL_NO_STDIO is
```
Comment (t8m) [inline doc/man3/OSSL_CMP_log_open.pod:102]: Should OPENSSL_NO_STDIO be in bold?

**Review by t8m — COMMENTED** [review]
Some nits

File: doc/man3/OSSL_CMP_ITAV_set0.pod (hunk 1 above)
Comment (levitte) [reply doc/man3/OSSL_CMP_ITAV_set0.pod:32]: Thanks, I'll fix them

File: doc/man3/OSSL_CMP_log_open.pod (hunk 2 above)
Comment (levitte) [reply doc/man3/OSSL_CMP_log_open.pod:102]: "Special macros, which are usually in uppercase, are in bold."

I think that yes, it should

File: doc/man3/OSSL_CMP_ITAV_set0.pod (hunk 1 above)
Comment (levitte) [reply doc/man3/OSSL_CMP_ITAV_set0.pod:32]: Done

File: doc/man3/OSSL_CMP_log_open.pod (hunk 2 above)
Comment (levitte) [reply doc/man3/OSSL_CMP_log_open.pod]: Done

**Review by t8m — APPROVED** [review]

**Comment by levitte** [comment]
Merged

f5f4fbaa44af055e0658c6810b91aa8607e8383a Make the OSSL_CMP manual conform with man-pages(7)

**Pull request closed without merging**